app/src/hooks/useGameDetailsQuery.ts

```diff
@@ -0,0 +1,22 @@
+import { Query, useQuery } from 'react-query';
+
+import { QUERY_GAMES } from '../const/query.const';
+import { useAxios } from './useAxios';
+import { MatchListResponse } from './useMatchListQuery';
+
+interface GameDetailsResponse extends MatchListResponse {
+  squads: object[]; //TODO: squad interface
+}
+
+export const useGameDetailsQuery = (match_id: number | null | undefined) => {
```

Comment:
samo `number | undefined` powinno wystarczyć - jeżeli poprzedni hook się jeszcze nie wykonał to `data` zwróci undefined, a jeżeli już się wykona to czy zakładamy, że zwrócony przez backend match_id może mieć wartość `null`?